chalice/app.pyi

```diff
@@ -109,6 +109,7 @@ class Chalice(object):
     api = ... # type: APIGateway
     routes = ... # type: Dict[str, Dict[str, RouteEntry]]
     current_request = ... # type: Request
+    lambda_context = ... # type: Any
```

Comment:
Can y we at least put a TODO here to type this properly once we add `local` support?